core/ledger/kvledger/txmgmt/privacyenabledstate/common_storage_db.go

```diff
@@ -283,7 +296,7 @@ func (s *CommonStorageDB) GetPrivateDataMetadataByHash(namespace, collection str
 // and the errors because of bad index files - the later being unfixable by the admin. Note that the error suppression
 // is acceptable since peer can continue in the committing role without the indexes. However, executing chaincode queries
 // may be affected, until a new chaincode with fixed indexes is installed and instantiated
-func (s *CommonStorageDB) HandleChaincodeDeploy(chaincodeDefinition *cceventmgmt.ChaincodeDefinition, dbArtifactsTar []byte) error {
+func (s *CommonStorageDB) HandleChaincodeDeploy(chaincodeDefinition *cceventmgmt.ChaincodeDefinition, dbIndexArtifactsTar []byte) error {
```

Comment:
Reverted the changes. I agree that the name has become a little longer. 

My thought process is that code for today while keeping the structure in a way that extensions are possible tomorrow. Though the `dbArtifactsTar` enables the easy extension, I am not sure whether we need this much ease. Looks like the `dbArtifactsTar` was added in May 2018. We haven't supported any new artifacts and I also doubt whether we support anything new going forward. As it only holds index, I renamed it. Further, I had a problem with `GetDBType()` too as we support index only for the CouchDB. I am sure that the probability of supporting a new DB is too low. I would prefer to be explicit in the code so that it would not confuse the reader like _what other artifacts are there?_ As it is unrelated to this PR, I am reverting the changes made.